src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_help.py

```diff
@@ -31,26 +31,17 @@
     type: command
     short-summary: Upload a file to a storage blob.
     long-summary: Creates a new blob from a file path, or updates the content of an existing blob with automatic chunking and progress notifications.
-    examples:
-        - name: Upload to a blob.
-          text: az storage blob upload -f /path/to/file -c MyContainer -n MyBlob
```

Comment:
The command has a lot of arguments so this one is still useful I think.

Reply:
I don't think that 3 arguments is a lot, but you are correct that we do need some better examples here. Any suggestions based on usage patterns you see?